pkg/kafka/endpoint_manager_test.go

```diff
@@ -77,7 +78,8 @@ func (ep *EPTest) initBackend() error {
 		return status.Error(codes.Internal, "Embedded server failed to start")
 	}
 
-	ep.backend = db.NewBackend("etcd", "127.0.0.1", kvClientPort, int(timeout.Milliseconds()), "service/voltha")
+	addr := kvstore.GetAddress("127.0.0.1", kvClientPort)
```

Comment:
Why introduce a addr variable. Use kvstore.GetAddress("127.0.0.1", kvClientPort) directly as second argument of db.NewBackend method